File: DataStructuresAndLibraries/__1000__/1162/1162.js
const fs = require('fs')
const input = fs.readFileSync('/dev/stdin', 'utf-8')
const lines = input.split('\n')

const cases = Number(lines.shift())

for(let i = 0; i < cases; i++){

    let len = Number(lines.shift())
    let vet = lines.shift().split(' ').map(Number)
    let count = 0

    //implementando o algoritmo de bubble sort
    for(let j = 0; j < len - 1; j++){
        for(let k = 0; k < len - j - 1; k++){

            //swapping
            if(vet[k] > vet[k + 1]){
                count++
                let temp = vet[k]
                vet[k] = vet[k + 1]
                vet[k + 1] = temp
            }
        }
    }

    console.log(`Optimal train swapping takes ${count} swaps.`)
}

File: DataStructuresAndLibraries/__1000__/1162/1162.py
n = int(input())

while n > 0:
    n -= 1
    l = int(input())
    vet = input().split()

    swaps = 0
    for i in range(l):
        for k in range(i + 1, l):
            if(int(vet[i]) > int(vet[k])):
                vet[i], vet[k] = vet[k], vet[i]
                swaps += 1

    print(f"Optimal train swapping takes {swaps} swaps.")

    #TLE